client/components/marketing-survey/cancel-jetpack-form/style.scss

```diff
@@ -148,4 +152,9 @@ input[type='text'].form-text-input.jetpack-cancellation-survey__choice-item-text
 			}
 		}
 	}
+
+	&__error-text {
+		color: var( --studio-red-0 );
+		font-weight: bold;
```

Comment:
We usually use numbers for `font-weight` :)

Reply:
nice, thanks!